robottelo/constants/__init__.py

```diff
@@ -1791,6 +1791,7 @@ class Colored(Box):
 FOREMAN_SETTINGS_YML = '/etc/foreman/settings.yaml'
 
 FOREMAN_TEMPLATE_IMPORT_URL = 'https://github.com/SatelliteQE/foreman_templates.git'
+NOF_FOREMAN_TEMPLATES = 18
```

Comment:
I'd prefer merging both template constants into a single dict, like:

```
FOREMAN_TEMPLATES_GIT = {
    'URL': 'https://github.com/SatelliteQE/foreman_templates.git',
    'COUNT': {
        'PUPPET_ENABLED': 18,
        'PUPPET_DISABLED': 17,
    },
    ...
}
```
and to bring this further, i think they all should be rater moved to `conf` as settings. However, this would imply the need to merge all `FOREMAN_TEMPLATE_` constants and that is way out of scope of this PR, so let's quickly forget about it ;)